pkg/windows/utils.go

```diff
@@ -31,9 +33,98 @@ var (
 	}
 )
 
+var softwareRegistryKey = "Software\\Rancher"
+var softwareCalicoRegistryKey = softwareRegistryKey+"\\Calico"
+
+
+func ensureRegistryKey() error {
```

Comment:
There's a lot of complexity here just to store the last reboot time in a registry key. I guess this is how Calico does it, do we have to do it this way too? Would it be easier to just store the timestamp in a file somewhere in the agent dir? We don't store anything else in the registry as far as I know.

Reply:
Yes. I was trying to do as Calico. But we can store it somewhere else too.

Reply:
Yeah that makes sense. But it looks like we're not using the same registry key as calico anyway, so we might as well do something simpler.

Reply:
I moved it on a file now the check should be more simpler.